### ISSUE TITLE
Should scene markers be included?

### ISSUE THREAD

**Issue by WithoutPants** [stashapp/metadata-api-discuss#2]
As per title. Are we including scene markers in this?

I've been hesitant to add things like duration and resolution to the scene schema, since there may be multiple files pointing to the same scene with various qualities and modifications. Markers may be correct for one file, but incorrect for another. The simplest solution is to exclude it altogether, but I can also see the usefulness of including it. Any thoughts?

**Comment by ghost** [comment]
I think it would be a shame not to have scene markers. Duration is also really useful for fuzzy matching videos. Given a performer name and duration you could probably 9 out of 10 times match the correct scene. Durations also help weed out obvious mistakes. If your video is 1h10 and the db says 35m, then it's likely wrong.

I get what you're saying though, the most accurate would be to tie duration and checksums to a separate resource/file object, then you could add resolution as well. MusicBrainz does something similar with recording/track durations where tracks have individual durations but the recording has its own "official" duration which is usually derived from the tracks.

**Comment by Leopere** [comment]
Technically, it would be useful to attempt to break down a perceptual hash down to segments within recorded scene durations.  This would be useful for locating scenes that are in mashups compilations or other home videos.  This isn't something we need necessarily, however, maybe we can consider an increase of resolution of hashing sections of the video, based on community submitted scene markers.

**Comment by ghost** [comment]
Yeah that's true, it would be invaluable if we ever support dvds.
